packages/aws-rfdk/lib/deadline/lib/render-queue-ref.ts

```diff
@@ -243,6 +243,13 @@ export interface RenderQueueProps {
    */
   readonly vpcSubnets?: SubnetSelection;
 
+  /**
+   * The subnets into which to place the Application Load Balancer that is deployed.
+   *
+   * @default - One Private subnet from each AZ.
+   */
+  readonly vpcAlbSubnets?: SubnetSelection;
```

Comment:
nit: does it make more sense to name this `vpcSubnetsAlb`? Unfortunately I had created the original `vpcSubnets` which dictates the placement of the ASG instances without thinking about the ALB. The original `vpcSubnets` came from the convention used in the CDK's standard AWS construct library:

* [`Instance` construct's `vpcSubnets` prop](https://docs.aws.amazon.com/cdk/api/latest/docs/@aws-cdk_aws-ec2.Instance.html#vpcsubnets)
* [`AutoScalingGroup` construct's `vpcSubnets` prop](https://docs.aws.amazon.com/cdk/api/latest/docs/@aws-cdk_aws-autoscaling.AutoScalingGroup.html#vpcsubnets)

To continue with that convention as best as possible, I suggest we stick with `vpcSubnets` prefix and append `Alb` as a suffix. That way those using code-completion will see both show up and can pick/explore accordingly. What do you think though?

Reply:
Looks good - thanks